### ISSUE TITLE
Open new tab with random color.

### ISSUE THREAD

**Issue by vaibhav0320** [microsoft/terminal#12412]
# Description of the new feature/enhancement

Currently we have to manualy  assign color to a tab when we open one. If there is feature which can assign random color to a new tab then it will be a good. First it looks cool and second it easy easiar to navigate to tabs.

**Comment by zadjii-msft** [comment]
Sure. I'm down. Something like #9422 probably.

**Comment by Inrmanish** [comment]
see my pull req,
gd or not

**Comment by aaron-613** [comment]
+1

Any way to do this yet in the current or preview versions..?

**Comment by zadjii-msft** [comment]
Nope. We'll make sure to update this thread when there is. In the meantime, might I recommend hitting the 👍 button on this issue?

![image](https://user-images.githubusercontent.com/18356694/96140789-ef3c8880-0ec5-11eb-82e0-30ce4977b9a7.png)

Thanks!
